packages/fxa-react/configs/tailwind.js

```diff
@@ -63,24 +49,15 @@ module.exports = {
         100: '25rem',
       },
       inset: {
-        '1/2': '50%',
-        '-px': '-1px',
-        '-3': '-0.75rem',
-        '-50': '-11.50rem',
-        '-52': '-13rem',
-        0.5: '0.125rem',
-        3: '0.75rem',
-        6: '1.25rem',
-        10: '2.5rem',
+        50: '11.50rem',
         55: '13.75rem',
         ten: '10%',
-        full: '100%',
       },
       top: {
-        2.5: '0.6rem',
+        2.5: '0.6rem', // default is 0.625rem
       },
       left: {
-        3.5: '.87rem',
+        3.5: '.87rem', //default is 0.875rem
```

Comment:
If the defaults for top-2.5 and left-3.5 are close enough, we could remove this custom config.